content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,209 @@
+INTEGRATING MONGODB WITH JAVA
+
+MongoDB is a NoSQL database where records get stored in documents. It uses JSON syntax a JavaScript Object Notation to store data. When integrating MongoDB with java it uses a different syntax.
+In this article, you'll learn how to connect MongoDB with java using eclipse and learn CRUD (create, read, update and delete) operations using Java and MongoDB Java Driver. MongoDB is based on a document model written in C++, JavaScript, and python.
+
+### Table of Contents
+
+1. [Prerequisites](#prerequisites) 
+2. [MongoDB versus Relational Database Management System](#MongoDB-versus-Relational-Database-Management-System) 
+3. [Connecting MongoDB with Java](#Connecting-MongoDB-with-Java) 
+4. [Creating MongoDB collection](#Creating-MongoDB-collection) 
+5. [Insert data in the collection](#Insert-data-in-the-collection) 
+6. [Update data in the collection](#Update-data-in-the-collection) 
+7. [Delete data in the collection](#Delete-data-in-the-collection) 
+8. [Delete the collection](#Delete-the-collection) 
+9. [Eclipse versus Intellij Idea in MongoDB integration with java](#Eclipse-versus-Intellij-Idea-in-MongoDB-integration-with-java) 
+9. [Conclusion](#Conclusion) 
+
+
+### Prerequisites
+
+To follow along with this tutorial you'll need the following:
+- A basic understanding of the Java language
+- Eclipse IDE installed In your OS
+- MongoDB installed in your OS
+- MongoDB java connector
+- Java JDK
+
+### MongoDB versus Relational Database Management System
+
+- MongoDB is a nonrelational database while Relational Database Management System is a relational database.
+- MongoDB stores data using Json docs, while RDBMS stores data using rows and columns.
+- MongoDB uses collections while RDBMS uses tables.
+- No schema definition is required in MongoDB, but a clear schema is required in RDBMS.
+- Database in MongoDB is easy to scale, while in RDBMS options for scalability are more limited.
+- RDBMS is slower than MongoDB.
+
+### Connecting MongoDB with Java.
+
+In Eclipse, create a new project. Right-click on the file created, click on properties, and on the java build path, you'll add the MongoDB java connector that you downloaded and then click on AddExternalJARS to create a connection between MongoDB and java. Make sure the MongoDB server is running before using eclipse.

```

Comment:
This can be exclusively a new sub-topic. This is a little out of scope, but explaning them a bit more would help. Or atleast add reference links that talks about this.